src/glue/crawler.ts

```diff
@@ -61,39 +61,127 @@ export interface ICrawlerTarget {
+   */
   readonly deleteBehavior?: DeleteBehavior;
+  /**
+   * Description of the Crawler
+   */
   readonly description?: string;
+  /**
+   * Name of the Crawler
+   */
   readonly name?: string;
+  /**
+   * When crawling an Amazon S3 data source after the first crawl is complete, specifies whether to crawl the entire dataset again or to crawl only folders that were added since the last crawler run.
+   * 
+   * @see [AWS::Glue::Crawler RecrawlPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-crawler-recrawlpolicy.html)
+   */
   readonly recrawlBehavior?: RecrawlBehavior;
+  /**
+   * A {@link aws-glue.SecurityConfiguration | SecurityConfiguration } object to apply to the Crawler
+   */
   readonly securityConfiguration?: SecurityConfiguration;
+  /**
+   * For scheduled crawlers, the schedule when the crawler runs.
+   * 
+   * @see [AWS::Glue::Crawler Schedule](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-crawler-schedule.html)
+   */
   readonly scheduleExpression?: Schedule;
+  /**
+   * The prefix added to the names of tables that are created.
+   * 
+   * @see [AWS::Glue::Crawler](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-crawler.html#cfn-glue-crawler-tableprefix)
+   */
   readonly tablePrefix?: string;
+  /**
+   * A collection of targets to crawl.
+   * 
+   * @see [AWS::Glue::Crawler](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-crawler.html#cfn-glue-crawler-targets)
+   */
   readonly targets?: ICrawlerTarget[];
+  /**
+   * The update behavior when the crawler finds a changed schema.
+   * 
+   * @see [AWS::Glue::Crawler SchemaChangePolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-crawler-schemachangepolicy.html#cfn-glue-crawler-schemachangepolicy-updatebehavior)
+   */
   readonly updateBehavior?: UpdateBehavior;
 }
 
+/**
+ * Create a Crawler resource to pull information from the provided resource
+ * 
+ * @see [AWS::Glue::Crawler](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-crawler.html)
+ */
+
 export class Crawler extends Resource {
   // Internal properties
   private readonly _classifiers: string[] = [];
   private readonly _targets: ICrawlerTarget[] = [];
 
   // Input properties
-  public readonly configuration?: CrawlerConfiguration;
-  public readonly database?: Database;
-  public readonly deleteBehavior?: DeleteBehavior;
-  public readonly description?: string;
-  public readonly name?: string;
-  public readonly recrawlBehavior?: RecrawlBehavior;
-  public readonly scheduleExpression?: Schedule;
-  public readonly securityConfiguration?: SecurityConfiguration;
-  public readonly tablePrefix?: string;
-  public readonly updateBehavior?: UpdateBehavior;
+  /**
```

Comment:
Indentation here looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now